cmd/bosun/conf/notify.go

```diff
@@ -29,30 +29,42 @@ func (n *Notification) Notify(subject, body string, emailsubject, emailbody []by
 		go n.DoEmail(emailsubject, emailbody, c, ak, attachments...)
 	}
 	if n.Post != nil {
-		go n.DoPost([]byte(subject))
+		go n.DoPost([]byte(n.GetPayload(subject, body)))
 	}
 	if n.Get != nil {
 		go n.DoGet()
 	}
 	if n.Print {
-		go n.DoPrint(subject)
+		if n.UseBody {
+			go n.DoPrint("Subject: " + subject + ", Body: " + body)
+		} else {
+			go n.DoPrint(subject)
+		}
+	}
+}
+
+func (n *Notification) GetPayload(subject, body string) (payload string) {
```

Comment:
I'd have this function return `[]byte` instead of `string` so you don't need to convert it at the call.